src/EnergyPlus/Furnaces.cc

```diff
@@ -8258,8 +8217,6 @@ namespace Furnaces {
             } else if (ZoneSensLoadMetFanONCompOFF < TotalZoneSensLoad) {
                 CoolPartLoadRatio = 0.0;
                 thisFurnace.CompPartLoadRatio = 0.0; // compressor OFF
-                thisFurnace.WSHPRuntimeFrac = 0.0;
-                thisFurnace.CoolingCoilSensDemand = 0.0;
```

Comment:
@nealkruis TODO: This might be an accidental deletion. Will check back after walking through the rest of the code.